drivers/hmis/app/models/hmis/form/record_type.rb

```diff
@@ -68,6 +68,11 @@ def self.all
           owner_type: 'Hmis::Hud::YouthEducationStatus',
           processor_name: 'YouthEducationStatus',
         ),
+        new(
+          id: 'GEOLOCATION',
+          owner_type: 'Hmis::Hud::Enrollment',
+          processor_name: 'Geolocation',
```

Comment:
Processor doesn't exist yet - will be part of https://github.com/open-path/Green-River/issues/6541